code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -111,7 +111,7 @@ var/list/ai_list = list()
 
 	aiMulti = new(src)
 	aicamera = new/obj/item/device/camera/ai_camera(src)
-
+	eyeobj.addHear()
```

Comment:
How do you mean, just eyeobj.flags = HEAR or something?
There's this thing in addHear
`getFromPool(/mob/virtualhearer, src)`

I don't really understand, would I still need to do that somewhere as well?